python/ray/data/tests/test_operators.py

```diff
@@ -841,6 +838,34 @@ def yield_five(block_iter: Iterable[Block], ctx) -> Iterable[Block]:
     # 100 inputs -> 100 / 10 = 10 tasks -> 10 * 5 = 50 output blocks
     assert op._estimated_output_blocks == 50
 
+    # Test limit operator estimation
+    input_op = InputDataBuffer(make_ref_bundles([[i, i] for i in range(100)]))
+    op = LimitOperator(100, input_op)
```

Comment:
should we also include a test for the case where limit < number of rows? we currently cover `limit = num_rows` and `limit > num_rows`, but not this last case.

Reply:
In this case there are 2 rows per block, so we have limit < number of rows